### ISSUE TITLE
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema

### ISSUE THREAD

**Issue by christocracy** [appbaseio-apps/reactivemaps-starter-app#1]
```
Mac OS 10.12.4
$ node -v
16.10.2
$ npm -v
3.10.10
$ webpack-dev-server --version
webpack-dev-server 2.4.2
webpack 2.3.3
```

```
$ npm run start

> reactivemaps-starter-app@1.0.0 start /Users/chris/Documents/workspace/transistor/reactivemaps-starter-app
> npm-run-all --parallel dev webpack-dev-server


> reactivemaps-starter-app@1.0.0 webpack-dev-server /Users/chris/Documents/workspace/transistor/reactivemaps-starter-app
> webpack-dev-server --inline --hot --port 8000


> reactivemaps-starter-app@1.0.0 dev /Users/chris/Documents/workspace/transistor/reactivemaps-starter-app
> webpack -d --watch

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.module.noParse should be one of these:
   [RegExp] | RegExp | [string] | string
   Don't parse files matching. It's matched against the full resolved request.
   Details:
    * configuration.module.noParse[0] should be an instance of RegExp.
    * configuration.module.noParse should be an instance of RegExp.
    * configuration.module.noParse[0]: The provided value "ws" is not an absolute path!
    * configuration.module.noParse should be a string.
```

**Comment by siddharthlatest** [comment]
This is likely the result of us being on webpack 1.x. @metagrover can we move to webpack 2.x?

**Comment by metagrover** [comment]
Fixed here: 492dad274d02b80bb2263e3007815215615f1ed3

**Comment by christocracy** [comment]
Works, thanks.
